demo/locales/source.json

```diff
@@ -303,6 +303,10 @@
     "description": "The name of a configuration value.",
     "message": "Ignore [PROPER_NAME:HLS] Text Stream Failures"
   },
+  "DEMO_FORCE_HLS_FULL_SEGMENT_REQUEST": {
+    "description": "The name of a configuration value.",
+    "message": "Force [PROPER_NAME:HLS] to use full segment request"
```

Comment:
This should be the same as the message in en.json, just with the markup.

E.g. "Force Full Segment Request"